colicoords/cell.py

```diff
@@ -756,7 +756,7 @@ def calc_xc(self, xp, yp):
         a0, a1, a2 = self.coeff
         # xp, yp = xp.astype('float32'), yp.astype('float32')
         # Converting of cell spine polynomial coefficients to coefficients of polynomial giving distance r
-        a, b, c, d = 4 * a2 ** 2, 6 * a1 * a2, 4 * a0 * a2 + 2 * a1 ** 2 - 4 * a2 * yp + 2, 2 * a0 * a1 - 2 * a1 * yp - 2 * xp

```

Comment:
If i remember correctly this polynomial is from setting the derivative of the distance r (where r comes from pythagorian theorem) to zero?
So the factor two can be divided out?